hamlpy/parser/filters.py

```diff
@@ -1,7 +1,13 @@
 from __future__ import unicode_literals
+"""
+Core HamlPy filters.
+
+The implementation of these should match https://github.com/haml/haml/blob/master/lib/haml/filters.rb as closely as
+possible. Where we differ is that we don't compile Stylus, Coffeescript etc into CSS or Javascript - but place the
+content into suitable <style> and <script> that can be transformed later by something like django-compressor.
+"""
```

Comment:
I think you are right.. well sheesh, that seems like something we should fix. I don't even like that happening with per-process caching.